### PR TITLE
updating Spanish translation credits.

### PR DESCRIPTION
**Pull request by alphacentauri82**
Adding the Spanish translation team

### CODE REVIEW

**Comment by NataliaTepluhina** [comment]
@alphacentauri82 thanks for the PR and for the translation work! There are two minor issues with the PR:

1) Could you please move `Translation repo` above the list of maintainers?
2) I think every maintainer besides Leonel is mentioned twice 😅

**Comment by alphacentauri82** [comment]
Done!. hehe sorry about that!

File: README.md [hunk 1]
```diff
@@ -81,9 +81,26 @@ Russian translation is maintained by Translation Gang.
 
 ### Spanish
 
-Spanish translation is maintained by [1950Labs](https://1950labs.com) and Vue.js Montevideo ([Leonel More](https://twitter.com/leonelmore), [Sebastian Camacho](https://twitter.com/sxcamacho), and [Diana Rodriguez](https://vue.beingadev.rocks).
-
-* Translation Repo - [/1950Labs/vuejs.org](https://github.com/1950Labs/vuejs.org)
+* Translation Repo - [/1950Labs/vuejs.org](https://github.com/1950Labs/vuejs.org)	* Translation Repo - [/1950Labs/vuejs.org](https://github.com/1950Labs/vuejs.org)
```
Comment (NataliaTepluhina) [inline README.md]:
```suggestion
* Translation Repo - [/1950Labs/vuejs.org](https://github.com/1950Labs/vuejs.org)	
```
I think there is also a doubling here

File: README.md [hunk 2]
```diff
@@ -81,9 +81,26 @@ Russian translation is maintained by Translation Gang.
...
+
+Spanish translation is maintained by:
+
+[1950Labs](https://1950labs.com) & [Vue.js Montevideo](https://www.meetup.com/Montevideo-Vue-JS-Meetup/):	[1950Labs](https://1950labs.com) & [Vue.js Montevideo](https://www.meetup.com/Montevideo-Vue-JS-Meetup/):
```
Comment (NataliaTepluhina) [inline README.md]:
```suggestion
[1950Labs](https://1950labs.com) & [Vue.js Montevideo](https://www.meetup.com/Montevideo-Vue-JS-Meetup/):	
```
And here 😅

File: README.md (hunk 1 above)
Comment (alphacentauri82) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (alphacentauri82) [reply README.md]: And Done!

**Comment by alphacentauri82** [comment]
THis is it! hehe apologies.

**Pull request merged**